command-expansion-server/sql/commanding/tables/user_sequence.sql

```diff
@@ -0,0 +1,39 @@
+create table user_sequence (
+  command_dict_id integer not null,
```

Comment:
I think we should mirror our expansion behavior here and have this be the `authoring_command_dict_id`. We don't want to confine sequence usage to a single dictionary, but rather be able to hint to the UI what dictionary to use when loading it.

Reply:
+1

Reply:
Done